cli/pbjs.js

```diff
@@ -99,6 +99,8 @@ exports.main = function main(args, callback) {
                 "                   es6       ES6 wrapper (implies --es6)",
                 "                   closure   Just a closure adding to protobuf.roots (see -r)",
                 "",
+                "  --dependency     Specifies which version of protobuf to require. Accepts any valid module id",
+                "",
```

Comment:
Ideally, the argument should always be honored in case the user has a legitimate reason to override it, but the targets should have their defaults.